dotnet/src/SemanticKernel.IntegrationTests/CoreSkills/PlannerSkillTests.cs

```diff
@@ -0,0 +1,141 @@
+            .AddJsonFile(path: "testsettings.development.json", optional: true, reloadOnChange: true)
+            .AddEnvironmentVariables()
+            .AddUserSecrets<OpenAICompletionTests>()
+            .Build();
+    }
+
+    [Theory]
+    [InlineData("Write a poem or joke and send it in an e-mail to Kai.", "function._GLOBAL_FUNCTIONS_.SendEmail")]
+    public async Task CreatePlanWithEmbeddingsTestAsync(string prompt, string expectedAnswerContains)
+    {
+        // Arrange
+        AzureOpenAIConfiguration? azureOpenAIConfiguration = this._configuration.GetSection("AzureOpenAI").Get<AzureOpenAIConfiguration>();
+        Assert.NotNull(azureOpenAIConfiguration);
+
+        AzureOpenAIConfiguration? azureOpenAIEmbeddingsConfiguration = this._configuration.GetSection("AzureOpenAIEmbeddings").Get<AzureOpenAIConfiguration>();
+        Assert.NotNull(azureOpenAIEmbeddingsConfiguration);
+
+        IKernel target = Kernel.Builder
+            .WithLogger(this._logger)
+            .Configure(config =>
+            {
+                config.AddAzureOpenAICompletionBackend(
+                    label: azureOpenAIConfiguration.Label,
+                    deploymentName: azureOpenAIConfiguration.DeploymentName,
+                    endpoint: azureOpenAIConfiguration.Endpoint,
+                    apiKey: azureOpenAIConfiguration.ApiKey);
+
+                config.AddAzureOpenAIEmbeddingsBackend(
+                    label: azureOpenAIEmbeddingsConfiguration.Label,
+                    deploymentName: azureOpenAIEmbeddingsConfiguration.DeploymentName,
+                    endpoint: azureOpenAIEmbeddingsConfiguration.Endpoint,
+                    apiKey: azureOpenAIEmbeddingsConfiguration.ApiKey);
+
+                config.SetDefaultCompletionBackend(azureOpenAIConfiguration.Label);
+            })
+            .WithMemoryStorage(new VolatileMemoryStore())
+            .Build();
+
+        TestHelpers.ImportSampleSkills(target);
+
+        var emailSkill = target.ImportSkill(new EmailSkill());
+
+        var plannerSKill = target.ImportSkill(new PlannerSkill(target));
+
+        // Act
+        ContextVariables variables = new(prompt);
+        variables.Set(PlannerSkill.Parameters.ExcludedSkills, "IntentDetectionSkill,FunSkill");
+        variables.Set(PlannerSkill.Parameters.ExcludedFunctions, "EmailTo");
+        variables.Set(PlannerSkill.Parameters.IncludedFunctions, "Continue");
+        variables.Set(PlannerSkill.Parameters.MaxFunctions, "9");
+        variables.Set(PlannerSkill.Parameters.RelevancyThreshold, "0.77");
+        SKContext actual = await target.RunAsync(variables, plannerSKill["CreatePlan"]).ConfigureAwait(true);
+
+        // Assert
+        Assert.Empty(actual.LastErrorDescription);
+        Assert.False(actual.ErrorOccurred);
+        Assert.Contains(expectedAnswerContains, actual.Result, StringComparison.InvariantCultureIgnoreCase);
+    }
+
+    private readonly XunitLogger<object> _logger;
```

Comment:
Could you move these private members up with the other one at the top of the file?

Reply:
It feels like most of our standard is keeping private stuff at the bottom -- I'll move the others down here for consistency.